### ISSUE TITLE
Intervention recipe - intervention/meeting took place

### ISSUE THREAD

**Issue by gmoger-jisc** [jiscdev/xapi#293]
I want to open this for discussion in our next call so that we can ensure the recipes align with development work happening with our systems.  

I believe initially these we designed with an 'intervention case management' approach but our requirements are a bit more simple in that we only need to identify when an intervention has taken place.

I will work with Matt Cownie to identify the requirements and report back further.

**Comment by ds10** [comment]
Being review due to legal:
* Will possibly interactions
* Will lack the detail
* Don't want to be capturing the notes.

**Comment by gmoger-jisc** [comment]
Our thinking on this is to capture 'interactions' more broadly.  In this way we can say that the student has interacted with different staff or support services, which might come from a CRM, IT/Library helpdesk or other student support systems.  We will want to reach out to institutions soon to try to identify what the data might look like but I want to update this ticket in the process as it is looking like a separate recipe from interventions.

**Comment by gmoger-jisc** [comment]
Can we put something provisional together on this, encompassing the following field requirements:

- STUDENT_ID
- INTERACTION_TYPE - email, meeting etc
- REASON (or TOPIC?) - Attendance, Achievement, Activity, Mentoring etc
- DATE
- INTERACTED_WITH (i.e. staff)

Happy to take feedback on any sensible field names and definitions.  I think the list of REASONS need to be clearly defined and should align with the scope of data that we can use (i.e. academic related only).

**Comment by arc12** [comment]
I'm a bit sceptical of the suggestion of a pre-defined list of REASONS. I have money on there being about as many different taxonomies of reason as there are CRM/CaseManagement implementations...

Take attendance for example. Fine if there is stand-alone attendance monitoring, but what if the local approach is to use a basket of factors to create an engagement score? Or if a wide-spectrum predictive risk indicator is used including attainment also?

I suppose it *might* be possible to have categories like: intervention/personal/routine-pastoral/other

**Comment by gmoger-jisc** [comment]
Maybe it's just left open then and we're clear, for our implementation at least, that we're only looking at interactions which relate to their academic aims.  My objective is to just make things clear so that there is no concern from a privacy/GDPR perspective.

**Comment by ds10** [comment]
I've put something provisional in 1.1.1 that needs discussion and work.

I'm not sure how the statement should be structured, as an example I've set the objectType as an agent, so the statement would do something like Actor-Verb-Actor, with TYPE and REASONS in the context.

For the case we put ACTOR-VERB-ACTIVTY, with the activity being a 'case' because staff would manipulate 'cases'. The details of the student were in the context.

**Comment by gmoger-jisc** [comment]
I like the idea of objectType as agent but I think we will need to be more flexible to accommodate different types of interactions.  This might simply be with a student service where we are not able to identify a particular agent.  Would would still capture it in the same way?

**Comment by ds10** [comment]
No, the object is required, so perhaps ACTOR-VERB-ACTIVTY would be the way to go

**Comment by gmoger-jisc** [comment]
Yes agreed - can we spec something provision on that basis?
